src/diffusers/models/resnet.py

```diff
@@ -640,6 +640,229 @@ def forward(self, input_tensor, temb):
         return output_tensor
 
 
+class Mish(torch.nn.Module):
+    def forward(self, hidden_states):
+        return hidden_states * torch.tanh(torch.nn.functional.softplus(hidden_states))
+
+
+class Upsample3D(nn.Module):
+    def __init__(self, channels, use_conv=False, out_channels=None, name="conv"):
```

Comment:
```suggestion
    def __init__(self, channels, use_conv=False, out_channels=None):
```
I'd prefer to get rid of it